fullstop-jobs/src/main/java/org/zalando/stups/fullstop/jobs/iam/NoPasswordViolationWriter.java

```diff
@@ -25,16 +29,39 @@ public NoPasswordViolationWriter(ViolationSink violationSink) {
         this.violationSink = violationSink;
     }
 
-    public void writeViolation(String accountId, User user) {
-        log.info("Found IAM user {} that has a password in account {}", user.getName(), accountId);
+    public void writeNoPasswordViolation(String accountId, CSVReportEntry csvReportEntry) {
+        log.info("Found IAM user {} that has a password in account {}", csvReportEntry.getUser(), accountId);
         violationSink.put(
                 new ViolationBuilder()
-                        .withEventId("check-iam-user_" + user.getName())
+                        .withEventId("check-iam-user_" + csvReportEntry.getUser())
                         .withAccountId(accountId)
                         .withRegion(NO_REGION)
                         .withPluginFullyQualifiedClassName(NoPasswordsJob.class)
                         .withType(PASSWORD_USED)
-                        .withMetaInfo(singletonMap("user_name", user.getName()))
+                        .withMetaInfo(singletonMap("user_name", csvReportEntry.getUser()))
+                        .build());
+    }
+
+    public void writeRootUserViolation(String accountId, CSVReportEntry csvReportEntry) {
+        log.info("Found IAM root user: {} that has configuration problem in account: {}", csvReportEntry.getUser(), accountId);
+
+        Map<String, String> metaInfo = new HashMap<>();
+        metaInfo.put("account_id", accountId);
+        metaInfo.put("user", csvReportEntry.getUser());
+        metaInfo.put("arn", csvReportEntry.getArn());
+        metaInfo.put("is_password_enabled", String.valueOf(csvReportEntry.isPasswordEnabled()));
+        metaInfo.put("is_mfa_active", String.valueOf(csvReportEntry.isMfaActive()));
+        metaInfo.put("is_access_key_1_active", String.valueOf(csvReportEntry.isAccessKey1Active()));
+        metaInfo.put("is_access_key_2_active", String.valueOf(csvReportEntry.isAccessKey2Active()));
+
+        violationSink.put(
+                new ViolationBuilder()
+                        .withEventId("check-iam-user_" + csvReportEntry.getUser())
+                        .withAccountId("put here the account property")
```

Comment:
here should be configured the property to get the account id that should than solve this violations.